phys/module_diag_afwa.F

```diff
@@ -561,8 +561,9 @@ SUBROUTINE afwa_diagnostics_driver (   grid , config_flags     &
 
             ! Add 273.15 to lifted index per some standard I don't understand
```

Comment:
@thomasedds 
Thomas,
Let's wait to hear back from Jimy and Wei. If they are OK with lifted index in (K), then this entire block should be removed, instead of having a line that says
```
grid%afwa_lidx(i,j) = grid%afwa_lidx(i,j)
```

But again, let's wait to hear their opinions.